addons/web/static/src/views/fields/statusbar_time/statusbar_time_field.scss

```diff
@@ -0,0 +1,28 @@
+.o_field_statusbar_time {
+    @extend .o_field_statusbar;
+    .time-btn {
```

Comment:
by convention we prefix our classnames by `o_`. Here, it looks like it's a bootstrap class, but it isn't